customTests/azure_cuda_bandwidth.nhc

```diff
@@ -67,7 +67,7 @@ function check_cuda_bw()
 	error_htod="**Fail** The htod gpu_copy test failed to execute."
 	error_htod+="It exited with error code"
 	if ! catch_error "$exec_htod" "$error_htod"; then
-        return 1
+        return 0
```

Comment:
Again what's going on with the indenting here?

Reply:
will look into

Reply:
addressed